service/Makefile.am

```diff
@@ -75,8 +75,12 @@ EXTRA_DIST = .gitignore \
              geopmdpy/session.py \
              geopmdpy/topo.py \
              geopmdpy/system_files.py \
```

Comment:
system_files.py should not be in EXTRA_DIST since system_files.py.in is in EXTRA_DIST.  This is because system_files.py is generated by running configure.